app/models/amazon_agent_manager.rb

```diff
@@ -0,0 +1,393 @@
+require 'aws-sdk'
+require 'amazon_ssa_support'
```

Comment:
These 2 should not be at the top-level otherwise the amazon library will be front-loaded.  Please delay load these if possible.